tests/test_other.py

```diff
@@ -5378,6 +5378,9 @@ def test_emterpreter_advise(self):
     out = run_process([PYTHON, EMCC, path_from_root('tests', 'emterpreter_advise_funcptr.cpp'), '-s', 'EMTERPRETIFY=1', '-s', 'EMTERPRETIFY_ASYNC=1', '-s', 'EMTERPRETIFY_ADVISE=1'], stdout=PIPE).stdout
     self.assertContained('-s EMTERPRETIFY_WHITELIST=\'["__Z4posti", "__Z5post2i", "__Z6middlev", "__Z7sleeperv", "__Z8recurserv", "_main"]\'', out)
 
+    out = run_process([PYTHON, EMCC, path_from_root('tests', 'emterpreter_advise_synclist.c'), '-s', 'EMTERPRETIFY=1', '-s', 'EMTERPRETIFY_ASYNC=1', '-s', 'EMTERPRETIFY_ADVISE=1', '-s', 'EMTERPRETIFY_SYNCLIST=["_j","_k"]'], stdout=PIPE).stdout
+    self.assertContained('-s EMTERPRETIFY_WHITELIST=\'["_a", "_b", "_e", "_f", "_main"]\'', out)
```

Comment:
does this emit different output based on the SYNCLIST parameter?

otherwise this all looks good.

Reply:
I'm not sure what you're asking. I wrote a call tree, picked two leaf nodes to be sync functions, and then calculated the whitelist from that.

Reply:
Sorry, my concern is that this doesn't fully test the new feature - I worry the output of that test would be the same before this PR (and without the SYNCLIST param). But maybe I'm missing something?

Reply:
(Maybe I'm asking the wrong question, and it's ok the output would have been the same?)

Reply:
Without the SYNCLIST param (or without the actual patch) the whitelist is just `[]`, because none of the functions call actual async functions.

Reply:
I see, thanks!